test/SearchTest.py

```diff
@@ -9,17 +9,28 @@
 from selenium.webdriver.support.ui import WebDriverWait
 from selenium.webdriver.support import expected_conditions as EC
 from selenium.webdriver.common.keys import Keys
+from selenium.webdriver.common.by import By
+import time
 
 class SearchTest(MediawikiBase):
     def test_01(self):
         """Searches for a word, expects empty but error-free result."""
         home_url = "http://mardi-wikibase"
         self.loadURL(home_url)
         current_url = self.driver.current_url
-        self.getElementById('searchInput').clear()
-        self.getElementById('searchInput').send_keys("boe")
-        self.getElementById('searchInput').send_keys(Keys.ENTER)
+
+        vue = self.driver.find_elements(By.XPATH,"//div[@id='p-search']/a")
+        if vue:
```

Comment:
Could you elaborate in which cases vue is the expected case, and in which the other case and add a comment?

Reply:
Yes, I've added it in the comments. The Vector 2022 Skin uses the WVUI components (https://github.com/wikimedia/wvui). Other skins do not use this as far as I know.